Tools/cases_generator/generate_cases.py

```diff
@@ -220,10 +228,23 @@ def __init__(self, inst: parser.InstDef):
             effect for effect in inst.inputs if isinstance(effect, parser.CacheEffect)
         ]
         self.cache_offset = sum(c.size for c in self.cache_effects)
+
         self.input_effects = [
             effect for effect in inst.inputs if isinstance(effect, StackEffect)
         ]
+        manual = False
+        for effect in self.input_effects:
+            if effect.cond:
+                manual = True
```

Comment:
why do you call this 'manual'?

Reply:
Because the PUSH() and POP() operations are executed "manually" rather than using TOP() and friends combined with STACK_GROW()/SHRINK(). It's not a great name but I couldn't think of something better that wasn't a whole sentence.